app/services/UserServiceImpl.js

```diff
@@ -29,7 +29,30 @@ module.exports = function(userDao) {
     });
   };
 
+  const signin = function(userData) {
+    return new Promise((resolve, reject) => {
+      _userDao.signin(userData)
+        .then((user) => {
+          comparePassword(userData.password, user.password)
```

Comment:
Method change reflecting in 'signout' branch but will rename to be more semantic.